server/game/cards/01-Core/IsawaAtsuko.js

```diff
@@ -12,7 +12,7 @@ class IsawaAtsuko extends DrawCard {
                     effect: ability.effects.modifyBothSkills(1)
                 })),
                 ability.actions.cardLastingEffect(context => ({
-                    target: this.game.currentConflict.getCharacters(context.player.opponent),
+                    target: context.player.opponent ? this.game.currentConflict.getCharacters(context.player.opponent) : [],
                     effect: ability.effects.modifyBothSkills(-1)
                 }))
```

Comment:
Similar to previous comment. Don't solve this in implementation but in getCharacters() by return an empty array